src/test/datascience/notebook/helper.ts

```diff
@@ -213,11 +218,13 @@ export async function waitForKernelToGetAutoSelected(expectedLanguage?: string)
         if (!vscodeNotebook.activeNotebookEditor.kernel) {
             return false;
         }
-        if (!expectedLanguage) {
-            kernelInfo = '<No specific kernel expected>';
-            return true;
-        }
         if (isJupyterKernel(vscodeNotebook.activeNotebookEditor.kernel)) {
+            if (!expectedLanguage) {
+                kernelInfo = `<No specific kernel expected> ${JSON.stringify(
+                    vscodeNotebook.activeNotebookEditor.kernel.selection
```

Comment:
Moved the code so we can see what kernel is selected on CI (for logging puproses).
FYI - this is test code.